src/stubs.ts

```diff
@@ -171,9 +171,28 @@ export function stubComponents(
       return [
         createStub({
           name: 'transition',
+          type: {
```

Comment:
Maybe you can pass the real transition component here? So we don't have all props defined which might change in the future